server/local/mod.ts

```diff
@@ -0,0 +1,54 @@
+import { Handler } from "../../deps/std/http/server.ts"
+import { Env, parsePathInfo } from "../mod.ts"
+import * as f from "./factories.ts"
+import { LandingPage } from "./pages/mod.ts"
+
+export function handler(env: Env): Handler {
+  return async (req) => {
+    const url = new URL(req.url)
+    const path = url.pathname.slice(1)
+    if (path === "") return f.page(LandingPage())
+    const pathInfo = parsePathInfo(path)
+    if (pathInfo) {
+      const { vCapi, vRuntime, providerId, generatorId, filePath } = pathInfo
+      const generatorProviders = env.providers[generatorId]
+      if (generatorProviders) {
+        const provider = generatorProviders[providerId]
+        if (provider) {
+          if (vCapi) {
+            return await f.fiveHundred(
```

Comment:
```suggestion
            return f.fiveHundred(
```
(and similar along the rest)